src/main/java/stsjorbsmod/cards/cull/Shake.java

```diff
@@ -0,0 +1,47 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.VulnerablePower;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Shake extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Shake.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 0;
+    private static final int VULNERABLE = 1;
+    private static final int UPGRADE_PLUS_CARDS = 1;
+    private static final int CARDS_DRAWN = 2;
+
+    public Shake() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = CARDS_DRAWN;
+        urMagicNumber = baseUrMagicNumber = VULNERABLE;
+        exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
```

Comment:
For cards with effects that are conditional like this, the beta branch introduced a new bit of UI where the cards glow gold instead of blue when the "extra" effect is available; it would be good to implement that here based on whether any enemies are currently vulnerable. See Counterspell for an example.